modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/multiple/CheckboxMultipleDDMFormFieldValueAccessor.java

```diff
@@ -55,13 +61,24 @@
 	public JSONArray getValue(
 		DDMFormFieldValue ddmFormFieldValue, Locale locale) {
 
-		Value value = ddmFormFieldValue.getValue();
+		JSONArray optionsValuesJSONArray = getOptionsValuesJSONArray(
+			ddmFormFieldValue, locale);
 
-		if (value == null) {
-			return createJSONArray("[]");
+		Matcher matcher = _pattern.matcher(optionsValuesJSONArray.getString(0));
```

Comment:
@carolmariaabb, no caso do tamanho de `optionsValuesJSONArray` ser maior que 1, não deveríamos checar o padrão do label id para as outras posições também?

Reply:
@marcosapmf eu considerei que se quando você adiciona um campo com `options`, e o `option ID` for gerado automaticamente, vai ser gerado pra todos, e alem disso o campo é `readOnly`. Mas se tiver o caso que o campo não for `readOnly` ai tem que checar todos mesmo, tu sabe se esse caso pode acontecer?

Reply:
@carolmariaabb, eu acredito que seja possível do campo do `optionId` não ser `readOnly`. Se tu deres uma olhada no componente React do [Options](https://github.com/liferay/liferay-portal/blob/c1afa06249ed258818a2e5abacaaf37a5042cd10/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js#L382), tem uma prop chamada `keywordReadOnly` que trata justamente isso.

Reply:
@marcosapmf vou mudar isso então e faço um `push force`. Valeu!